Makefile

```diff
@@ -54,5 +54,5 @@ VERBOSE = 1
 MAKEFILE_PATH = ../asf-3.21.0/common/utils/make/Makefile.avr.in
 include $(MAKEFILE_PATH)
 
-program: all
```

Comment:
This should not be in the same PR. It's ok now, but for future PR, keep things separate.